rfcs/045-sierra-work-relationships/README.md

```diff
@@ -0,0 +1,377 @@
+https://search.wellcomelibrary.org/iii/encore/record/C__Rb2058813__Smontage__Ff:facetmediatype:k:k:Pictures::__P0,9__Orightresult__U__X3?lang=eng&suite=cobalt&marcData=Y
+) 
+
+```
+773 0  |tJames Gardiner Collection: Romantic fantasy and comic 
+       postcards, and photographs on gender and masculinity.|q3a 
+```
+
+These should be treated as a Series.
+
+```json
+"partOf": [
+  {
+    "title": "James Gardiner Collection: Romantic fantasy and comic postcards, and photographs on gender and masculinity.",
+    "type": "Series"
+  }
+]
+```
+
+In some cases, there is a 773 with a matching 830 or 4x0 property.
+This should result in only one partOf.
+
+```
+773 0  |tACLS Humanities E-Book.|nURL: http://
+           www.humanitiesebook.org/
+830  0 ACLS Humanities E-Book.
+```
+
+```json
+"partOf": [
+  {
+    "title": "ACLS Humanities E-Book.",
+    "type": "Series"
+  }
+]
+```
+
+A record may be part of multiple Series, e.g. [Derrida after the end of writing](https://search.wellcomelibrary.org/iii/encore/record/C__Rb3180411__S%20Fordham%20perspectives%20in%20continental%20philosophy__Orightresult__U__X3?lang=eng&suite=cobalt&marcData=Y):
+
+```
+773 0  |tACLS Humanities E-Book.|nURL: http://
+       www.humanitiesebook.org/ 
+830  0 Fordham Perspectives in Continental Philosophy. 
+830  0 ACLS Humanities E-Book. 
+```
+
+This will result in a partOf entry for each series.
+
+As there is no object corresponding to the parent, there is no symmetrical
+`parts` list.
+
+### In Depth
+
+A Series may be something like:  
+
+* a run of books, possibly, but not necessarily, on a common subject.
+  * e.g. the Usborne Touchy-Feely series, or Oxford Very Short Introductions
+* A grouping of things, possibly from an external source.
+
+These are designated using the 4xx and 8xx codes, and (normally) only identified by name.
+
+These should not be represented in the same fashion as a CALM hierarchy because:
+
+* There may be very many objects in any given series, 
+* There may not be any inherent order, the series is just a bag of items.
+
+However, membership of a series is useful information, as is the ability to 
+find other objects from that series.
+
+#### series title and subPart
+
+Separating the subPart from the series title allows us to create an accurate
+filtered search. The partOf title is the exact match to use.
+
+##### Why not store it all in title?
+
+Storing the series title and subPart in the same field makes this difficult.
+
+One could expect there to be only one "Published papers (Wellcome Chemical Research Laboratories) ; no. 149",
+and at least 149 "Published papers (Wellcome Chemical Research Laboratories)". One would
+also expect there to be many Series with titles containing most of those tokens.
+
+* A search for all those tokens would return too many results (anything containing Published, or papers, or Wellcome etc.)
+* A search for the exact string would return only one result (i.e. this one)
```

Comment:
It may be that we end up presenting two links. I'm not sure what the filter would look like. Perhaps we would index the whole field as one, but return the two separately in the API.

Reply:
You'll note that Encore only searches for `Early European Books : Printed sources to 1700` and doesn't include the `Collection 15`. I'm not saying it might not be _better_ to allow searching for both, but I do think that it's not necessary at this stage.

Volumes are often, but not always, a meaningless construct when it comes to research as they are primarily an organisational construct to help the publisher. That's the "how do we know problem", so I'd agree we make this a problem for the future, should it prove to be one.